client/src/components/Error/Error.js

```diff
@@ -1,13 +1,25 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import errorToast from '../utils/Toasts/default';
+import './Error.css';
 
 class Error extends Component {
+    componentDidMount() {
+        var { location } = this.props;
+        // Toast the error message
+        if (location.state && location.state.message) {
+            toast(location.state.message);
+        }
```

Comment:
Our history object sent over a message for the Error component to display, so display the error.